platform/packages/currency/src/test/group.rs

```diff
@@ -164,6 +177,15 @@ impl Group for SubGroup {
     const DESCR: &'static str = "sub_group";
     type TopG = SuperGroup;
 
+    fn currencies() -> impl Iterator<Item = CurrencyDTO<Self>> {
+        [
+            SubGroupTestC6::definition().dto(),
+            SubGroupTestC10::definition().dto(),
+        ]
+        .into_iter()
+        .map(ToOwned::to_owned)
```

Comment:
Please refer to previous comment.
```suggestion
        .copied()
```